src/select2/choices.tpl.html

```diff
@@ -1,4 +1,5 @@
-<ul class="ui-select-choices ui-select-choices-content select2-results">
+<ul class="ui-select-choices ui-select-choices-content select2-results"
+    ng-style="{top: position.top+'px', left: position.left+'px'}">
```

Comment:
I had the impression that you were making this directive "opt-in", meaning that it only impacts the directive if invoked.  This gives me the opposite impression, can you explain a bit for me?